packages/components/inputs/selectable-search-input/src/selectable-search-input.tsx

```diff
@@ -0,0 +1,434 @@
+   */
+  onChange?: (event: TCustomEvent) => void;
+  /**
+   * Called when input is blurred
+   */
+  onBlur?: (event: TCustomEvent) => void;
+  /**
+   * Called when input is focused
+   */
+  onFocus?: (event: TCustomEvent) => void;
+  /**
+   * Handler when the search button is clicked.
+   */
+  onSubmit: (searchValue: string) => void;
+  /**
+   * Handler when the clear button is clicked.
+   */
+  onReset?: () => void;
+  /**
+   * Focus the input on initial render
+   */
+  isAutofocussed?: boolean;
+  /**
+   * Indicates that the input cannot be modified (e.g not authorized, or changes currently saving).
+   */
+  isDisabled?: boolean;
+  /**
+   * Indicates that the field is displaying read-only content
+   */
+  isReadOnly?: boolean;
+  /**
+   * Indicates if the input has invalid values
+   */
+  hasError?: boolean;
+  /**
+   * Indicates if the input has warning values
+   */
+  hasWarning?: boolean;
+  /**
+   * Placeholder text for the input
+   */
+  placeholder?: string;
+  /**
+   * Indicates if the input should be cleared when the clear button is clicked.
+   * Defaults to true.
+   *
+   */
+  isClearable?: boolean;
+  /**
+   * Horizontal size limit of the input fields.
+   */
+  horizontalConstraint?:
+    | 3
+    | 4
+    | 5
+    | 6
+    | 7
+    | 8
+    | 9
+    | 10
+    | 11
+    | 12
+    | 13
+    | 14
+    | 15
+    | 16
+    | 'scale'
+    | 'auto';
+  /**
+   * Array of options that populate the select menu
+   */
+  options: TOption[] | TOptionObject[];
+  /**
+   * z-index value for the menu portal
+   */
+  menuPortalZIndex?: number;
+  /**
+   * Dom element to portal the select menu to
+   * <br>
+   * [Props from React select was used](https://react-select.com/props)
+   */
+  menuPortalTarget?: ReactSelectProps['menuPortalTarget'];
+  /**
+   * whether the menu should block scroll while open
+   * <br>
+   * [Props from React select was used](https://react-select.com/props)
+   */
+  menuShouldBlockScroll?: ReactSelectProps['menuShouldBlockScroll'];
+  /**
+   * Handle change events on the menu input
+   * <br>
+   * [Props from React select was used](https://react-select.com/props)
+   */
+  onMenuInputChange?: ReactSelectProps['onInputChange'];
+  /**
+   * Can be used to render a custom value when there are no options (either because of no search results, or all options have been used, or there were none in the first place). Gets called with { inputValue: String }.
+   * <br />
+   * `inputValue` will be an empty string when no search text is present.
+   * <br>
+   * [Props from React select was used](https://react-select.com/props)
+   */
+  noMenuOptionsMessage?: ReactSelectProps['noOptionsMessage'];
+  /**
+   * Whether to enable search functionality.
+   * <br>
+   * [Props from React select was used](https://react-select.com/props)
+   */
+  IsMenuSearchable?: ReactSelectProps['isSearchable'];
```

Comment:
Should this be lowercase? 🤔

Reply:
fixed here [4bae45c](https://github.com/commercetools/ui-kit/pull/2426/commits/4bae45c97abfcdf9ff54bbfff2bcde28afa5e400)